external/fv3fit/fv3fit/emulation/thermobasis/emulator.py

```diff
@@ -156,6 +157,11 @@ def register_parser(parser):
             help="comma separated list of variable names.",
         )
 
+        # other options
+        parser.add_argument(
```

Comment:
nit: Is there a way to generate the args from the dataclass or vice-versa? The data here is currently duplicated which can become a source of bugs.

Reply:
This is a good idea for future work.